src/views/CavaticaBixView.js

```diff
@@ -0,0 +1,197 @@
+import React, {useState} from 'react';
+import {graphql, compose} from 'react-apollo';
+import {GET_STUDY_BY_ID, MY_PROFILE, GET_PROJECTS} from '../state/queries';
+import {SYNC_PROJECTS, LINK_PROJECT, UNLINK_PROJECT} from '../state/mutations';
+import {
+  Container,
+  Segment,
+  Message,
+  Placeholder,
+  Header,
+  Button,
+} from 'semantic-ui-react';
+import EmptyView from './EmptyView';
+import NewProjectModal from '../modals/NewProjectModal';
+import LinkProjectModal from '../modals/LinkProjectModal';
+import CavaticaProjectList from '../components/CavaticaProjectList/CavaticaProjectList';
+
+const CavaticaBixView = ({
+  study: {loading, studyByKfId, error},
+  projects: {allProjects, projectLoading, projectError},
+  user,
+  syncProjects,
+  linkProject,
+  unlinkProject,
+}) => {
+  const isAdmin = !user.loading
+    ? user.myProfile.roles.includes('ADMIN')
+    : false;
+  const [showModal, setShowModal] = useState('');
+
+  if (loading)
+    return (
+      <Container as={Segment} basic vertical>
+        <Placeholder>
+          <Placeholder.Header image>
+            <Placeholder.Line />
+            <Placeholder.Line />
+          </Placeholder.Header>
+          <Placeholder.Paragraph>
+            <Placeholder.Line />
+            <Placeholder.Line />
+            <Placeholder.Line />
+            <Placeholder.Line />
+          </Placeholder.Paragraph>
+        </Placeholder>
+      </Container>
+    );
+  if (error)
```

Comment:
You may want to check errors for the user and projects, too, not just the study errors.
You may have to concatenate them all together and check their result.